### ISSUE TITLE
jsdoc markdown rendering under @param partially broken

### ISSUE THREAD

**Issue by jasonkuhrt** [microsoft/vscode#112292]
Given this jsdoc block:
```
/**
 * 1. As string literals matching the name of a builtin [scalar](https://foo.bar).  
 *   
 *   E.g.: 'ID', 'String', ...
 *
 * @param type Blah blah
 *
 * 1. As string literals matching the name of a builtin [scalar](https://foo.bar).  
 *   
 *   E.g.: 'ID', 'String', ...
 */
```
The following is rendered:

![image](https://user-images.githubusercontent.com/284476/101917278-0df18f80-3b96-11eb-8a47-d8c7c70248d5.png)

Notice the list indentation is lost under `@param`. But other markdown works like the link.

**Comment by rafaelrozon** [comment]
I would like to try to fix this one.

**Comment by CreaTorAleXander** [comment]
is it in progress @rafaelrozon? Otherwise I would like to take it.

**Comment by rafaelrozon** [comment]
@CreaTorAleXander, yes. It's in progress.

**Comment by chenjigeng** [comment]
I think this might have something to do with quickinfo for typescript @rafaelrozon

**Comment by rafaelrozon** [comment]
Sorry, guys. I won't be able to get this working. I had some personal problems and I won't have the time to look into this. Sorry for any inconvenience.

**Comment by alexlarbi** [comment]
I would like to give it a shot if no one else is working on it.

**Comment by hantatsang** [comment]
@jasonkuhrt I tried having a go at this but found out this is not a problem with the Markdown renderer nor with VSCode. That's just how JSDoc parsing works - it doesn't like having tags (like `@param` in this case) to be in a list. Tags are treated differently so it shouldn't be part of a Markdown block that you're writing.

I haven't gone as far as inspecting VSCode's jsdoc parsing but I've tested out a number of other IDE/tools, the rendered documentation for your example will all break:

1. Jetbrains Web-Micro:

    ![screenshot-web-micro](https://user-images.githubusercontent.com/11912225/105706463-ae90e880-5f65-11eb-9147-0e797fd0a4bd.jpg)

1. [JSDoc Live Preview](https://marketplace.visualstudio.com/items?itemName=Perkovec.jsdoc-live-preview)

    ![screenshot-jsdoc](https://user-images.githubusercontent.com/11912225/105706471-b05aac00-5f65-11eb-91ad-86b33e9f477c.jpg)
